### PR TITLE
feat :added snackbar component

### PR DESCRIPTION
**Pull request by developer-anand**
image

### CODE REVIEW

File: components/snackbar/snackbar.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  position: fixed;
+  z-index: 1;
+  align-items: center;
+}
+
+#snackbar.show {
+  visibility: visible; /* Show the snackbar */
+  /* Add animation: Take 0.5 seconds to fade in and out the snackbar.
+    However, delay the fade out process for 2.5 seconds */
+  -webkit-animation: fadein 0.5s, fadeout 0.5s 2.5s;
```
Comment (Shubhamkr2610) [inline components/snackbar/snackbar.css:48]: Appreciation: its good to give message . I liked it

File: components/snackbar/snackbar.html [hunk 2]
```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>modal</title>
```
Comment (Shubhamkr2610) [inline components/snackbar/snackbar.html]: you should replace modal with snack bar

File: components/snackbar/snackbar.html [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+        </div>
+     
+      </div>    
+      <script>
+          function showSnackbar() {
+             const toast= document.getElementById("snackbar");
+             toast.className = "show";
+             setTimeout(function(){ toast.className = toast.className.replace("show", ""); }, 5000);
+}
+      </script>
```
Comment (Shubhamkr2610) [inline components/snackbar/snackbar.html:177]: avoid using inline js .....I think it will be better to move it in a .js file

**Review by Shubhamkr2610 — APPROVED** [review]
Looks good to merge

**Review by Muzammil-khan710 — APPROVED** [review]
Clean code 
Awesome UI
after changing the above stated suggestions...
it will be good to merge

File: components/snackbar/snackbar.html (hunk 2 above)
Comment (developer-anand) [reply components/snackbar/snackbar.html]: thanks for pointing this

File: components/snackbar/snackbar.css [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  align-items: center;
+  margin: 2rem;
+}
+
+#snackbar {
+  visibility: hidden;
+  top: 50rem;
+  min-width: 25rem;
+  margin-left: -13rem;
+  background-color: var(--black);
```
Comment (varun13169) [inline components/snackbar/snackbar.css:34]: Name color more descriptively instead of just black

File: components/snackbar/snackbar.css [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+  color: var(--secondary-color);
+  text-align: center;
+  border-radius: 2px;
+  padding: 2rem;
+  position: fixed;
+  z-index: 1;
```
Comment (varun13169) [inline components/snackbar/snackbar.css:40]: Avoid Z index, but ok if you want to continue with this.

**Review by varun13169 — APPROVED** [review]

**Review by sanu29 — APPROVED** [review]
Good to merge

**Pull request merged**